metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -367,12 +367,12 @@ def __init__(self, config: SQLCommonConfig, ctx: PipelineContext, platform: str)
             )
 
     def warn(self, log: logging.Logger, key: str, reason: str) -> None:
-        self.report.report_warning(key, reason)
+        self.report.report_warning(key, reason[:100])
```

Comment:
Alternatively, cutting the reason message could be done in the `report_warning` method https://github.com/datahub-project/datahub/blob/85fa5a1c4fdf2b4c4439558fa3a4cbbfd3491fbf/metadata-ingestion/src/datahub/ingestion/api/source.py#L97 
To be decided by the reviewer 😅